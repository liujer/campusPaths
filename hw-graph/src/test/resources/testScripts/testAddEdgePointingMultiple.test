# Tests adding of edges by having one node point
# to multiple nodes
CreateGraph D
AddNode D one
AddNode D two
AddNode D three
AddEdge D one two e12
AddEdge D one three e13
ListNodes D
ListChildren D one
ListChildren D two
ListChildren D three