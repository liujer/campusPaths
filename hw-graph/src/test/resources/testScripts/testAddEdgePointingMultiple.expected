# Tests adding of edges by having one node point
# to multiple nodes
Created graph D
Added node one to D
Added node two to D
Added node three to D
Added edge e12 to D
Added edge e13 to D
ListNodes of D output: one three two
the children of one in D are: three(e13) two(e12)
the children of two in D are:
the children of three in D are: